sources/packages/web/src/components/students/applicationTracker/ApplicationProgressBar.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <template v-if="applicationStatus !== ApplicationStatus.cancelled">
+    <body-header title="Track your application" />
+    <v-slider
```

Comment:
I would preferer to have only the `v-slider` in the `ApplicationProgressBar` component.